hydra-node/test/Hydra/HeadLogicSpec.hs

```diff
@@ -279,7 +286,7 @@ spec = do
           _ -> False
         s1 <- assertNewState outcome1
         let oneSecondsPastDeadline = addUTCTime 1 contestationDeadline
-            stepTimePastDeadline = OnChainEvent $ Tick oneSecondsPastDeadline
+            stepTimePastDeadline = OnChainEvent $ Tick oneSecondsPastDeadline (ChainSlot 1)
```

Comment:
Could: use a weirder number than 1 or ``someSlot = arbitrary `generateWith` 42`` to indicate it should not matter which we choose